google-cloud-spanner/src/main/java/com/google/cloud/spanner/AsyncTransactionManagerImpl.java

```diff
@@ -54,6 +54,9 @@ public void setSpan(Span span) {
 
   @Override
   public void close() {
+    if (txnState == TransactionState.STARTED) {
+      rollbackAsync();
```

Comment:
Can there be a version of `close()` that propagates the `ApiFutrue` from `rollbackAsync()` back to the caller? Otherwise the application could shut down (or, in my case, the test terminate) before the rollback happens.

Reply:
I've added a `closeAsync()` method to the `AsyncTransactionManager` interface that returns an `ApiFuture` that is done when everything has been rolled back and released. PTAL.